<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>PEST RESISTANT</title>

    <style>


  #navbar {
  background: rgba(20, 80, 25, 5);
  color: #fff;
  overflow: auto;
  font-family: cursive;
  }

  #navbar a {
  color: rgb(221, 221, 233);
  text-decoration: none;
  }

  #navbar h1 {
  float: left;
  padding-top: 10px;
  }

  #navbar ul {
  list-style: none;
  float: right;
  }

  #navbar ul li {
  float: left;
  }

  #navbar ul li a {
  display: block;
  padding: 20px;
  text-align: center;
  }

  #navbar ul li a:hover, 
  #navbar ul li a.current {
    background: rgba(1, 126, 78, 0.26);
  color: rgb(74, 233, 153);
  }

  .container {
  margin: auto;
  max-width: 1100px;
  overflow: auto;
  padding: 0 20px;
  }

  a{text-decoration: none;}

  #main-footer {
  text-align: center;
  background: green;
  color: #fff;
  padding: 20px;
}

.main{
  color: whitesmoke;
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: cursive;
  height: 35rem;
  width: 100%;
}

#scroll{
    width: 70%;
    height: 500px;
    background-color: rgba(221, 191, 133, 0);
    border: 1px solid silver;
    margin-top: 20px;
    box-shadow: 1px 2px 5px 5px greenyellow;
    border-radius: 10px;
    opacity: 1;
    backdrop-filter: blur(15px);
    overflow: hidden;
    overflow-y: scroll;
    position: relative;
    padding: 5px;
    
}

body{
  background-image: url(../PICS/pests\ \(3\).jpg);
  background-repeat: no-repeat;
  background-size: cover;
}

     
</style>
</head>
<body>

  <header>
    <nav id="navbar">
      <div class="container">
        <h1 class="logo" alt="Fast Biotechnological Crops"><a href="INDEX.HTML">FBC</a></h1>
        <ul>
          <li><a href="INDEX.HTML">HOME</a> </li>
          <li><a href="GALLERY.HTML">GALLERY</a> </li>
          <li><a href="ABOUT.HTML">ABOUT</a> </li>
        </ul>
      </div>  
    </nav>
  </header>

  <div class="main">
    <div id="scroll">
  <h2>PEST RESISTANT</h2>
  <p>
    Plants and insects have co-existed for at least 400 million years. During this time, plants have evolved numerous strategies to attract insects as pollinators, while deterring them from consuming the plant. This ongoing battle between plants and insects has led to the development of multiple defensive and offensive approaches by both insects and plants. Plants produce a multitude of constitutive and inducible defense barriers to reduce damage by insects. They include physical barriers such as waxy layers and trichomes as well as molecular barriers, ranging from secondary metabolites to proteins.
    One exquisite example of this finely tuned plant-insect interaction is the defensive function of proteinase inhibitors (PIs) in solanaceous plants. Plants respond to insect attack by upregulating the expression of PIs in stems and leaves. The PIs interfere with protein digestion, which results in growth retardation or death of the insect pest. The struggle between plants and insects to survive and thrive has driven the diversification and expansion of gene families encoding both plant PIs and the target insect proteinases.
    Regulation of PIs in plant defense
    Clarence Ryan and colleagues were the first to characterize genes that encode potato type I and potato type II serine proteinase inhibitors and to discover an intriguing level of complexity that communities of plants use against insect attack (reviewed by 1). For example, when a caterpillar chews on tobacco or tomato plant leaves, a local signal derived from the insect or plant2 stimulates the expression of PIs and other defense related compounds (Fig. 1). The plant then produces a small mobile peptide called systemin that moves systemically via phloem from wounded to unwounded leaves, prearming unwounded leaves before they are damaged by the insect. Furthermore, the plants then produce a volatile molecule, methyl jasmonate, that travels to neighboring unwounded plants and triggers the signaling pathway that activates PI genes and heightens the defenses of the whole plant population3.
    Baldwin and colleagues have dissected various components of plant defense signaling pathways that lead to production of jasmonic acid (JA) and accumulation of proteinase inhibitors4-5. An antisense
    approach was used to knockdown expression of various upstream genes that
blocked production of JA. In one example, transcription factors from the WRKY
family, which mediate direct and indirect defenses in Nicotiana attenuate, were
silenced, severely reducing the levels of JA in response to wounding by insects6.
Accumulation of proteinase inhibitors and volatile alarm signals was significantly
diminished in these plants, resulting in substantial herbivore damage when grown in
their natural environment. When the defense response was reinstated by treatment
with exogenous JA, PI production was restored and Manduca sexta larvae feeding
on these plants exhibited the same slow growth as larvae feeding on native plants.
Given that knocking out a signaling pathway in these plants may have caused
changes other than PI production that might affect insect growth, Baldwin and
coworkers went further and demonstrated that the endogenous proteinase inhibitors
were directly responsible for reduced herbivore performance7.
Application of PIs for plant protection
Early observations linking the induction of PI gene expression to reduced insect
fitness inspired researchers to investigate the potential ability of PIs to protect
transgenic plants against herbivory from insect pests. As early as 1987, transgenic
plants expressing heterologous proteinase inhibitors were reported to be more
resistant to damage caused by chewing insects; there are now numerous examples
of transgenic plants expressing PIs with improved protection against insects
(reviewed by8-9). Although PIs have great potential, there are still no transgenic
plants expressing PIs in commercial development.
A number of parameters have been identified that influence the effectiveness of
PIs. These include insect gut pH, larval developmental stage, quantity and quality
of dietary protein, and concentration of PI. Another important factor leading to
the failure of PIs to provide adequate protection against insects has been the use
of single PIs directed to a subset of digestive proteinases, which leaves other
proteases unaffected and thus fails to provide sustainable plant protection. The
application of PIs in crop protection depends on a better understanding of how
insects respond and adapt to PIs.
PI mechanism of action
The antimetabolic effect of PIs on insects is not completely understood, but there
are several responses that have been described. 1) Growth and development of
insect larvae is severely retarded without significant changes in the gut proteolytic
activity. This observation led to the theory that hyperproduction of proteases would
in turn cause depletion of essential amino acids. 2) Growth and development of
insect larvae is severely retarded and is associated with a severe reduction in
proteolytic activity. In this case, amino acids for growth and development are
limited, due to the lack of processing of dietary protein by digestive proteases.
On the other hand, insects can employ a large number of mechanisms to
overcome the negative effects of proteinase inhibitors. There are several adaptive
mechanisms described: 1) upregulation of proteinases to compensate for inhibition
of another group of proteinases; 2) production of proteinases that degrade PI; and
3) production of PI-insensitive proteases. Insects can use a combination of these
mechanisms to maintain sufficient proteolysis for growth and development.
It is often difficult to determine which of these
adaptive processes is operating, because most
research articles only describe changes in gut enzyme
activity or gene transcription levels after PI ingestion.
The changes in levels of enzyme activity could have
resulted from PI binding to the enzyme, reduction in
gene expression or enzyme secretion, or increased
degradation of proteases. Understanding how insects
respond to PIs on a case-by-case basis will provide
valuable knowledge that may enable the development
of PIs as a viable transgenic plant protection
technology.
Controlling Lepidopteran pests
In a recent paper10, we described which of these
adaptive mechanisms operate in the highly destructive
insect pests Helicoverpa armigera and H. punctigera
when challenged with a series of serine proteinase
inhibitors from the flowers of the ornamental tobacco
Nicotiana alata (NaPI). These insects belong to the
infamous insect order Lepidoptera, which are major
pests of more than 100 species of crop plant worldwide.
In 2001 the global cotton market alone was worth 20
billion dollars, and yield loss combined with the cost
of controlling lepidopteran pests was estimated at $3
billion dollars annually11.
H. armigera species have been difficult to
control due to development of resistance to several
major insecticides. This problem was addressed
by the commercialization of transgenic crop plants
expressing Bt toxin, which also reduced use of
pesticides to control lepidopteran species. Insecticide
use fell by half in Indian cotton production and by
85% in Australian cotton production with introduction
of Bt cotton. Although use of Bt toxin genes has been
a highly successful technological advance to control
lepidopteran pests, and more than 58 million hectares
of Bt crops were grown globally in 2010, there is
evidence that field populations of insects are beginning
to develop resistance. Until now, this resistance has
not caused any major crop losses, but the proliferation
of resistance to Bt remains a real threat to global food
production. A continued effort to discover insecticidal
molecules with different targets to Bt is critical for
sustainable protection of crops against insect pests.
The Na-proPI multidomain potato type II inhibitor
and its application in crop protection
The potato inhibitor II (PinII) family of serine proteinase
inhibitors is widely distributed in Solanaceae. This
family arose from gene duplication of an ancestral gene
encoding a single proteinase inhibitor domain. The most
well known members of this family are two domain
inhibitors from potato and tomato, although members
with one, three, four, six, and eight proteinase inhibitor
domains have been described. NaPI is a 40.3 kDa protein
composed of an N-terminal endoplasmic reticulum signal
peptide that precedes six 6 kDa proteinase inhibitor
domains followed by a C-terminal vacuolar-targeting
signal (VTS). Unusual folding of PI domains results
in a circular bracelet-like structure clasped by three
disulfide bonds between N- and C-terminal fragments
(Fig. 2). Processing NaPI in the secretory pathway
removes the signal peptide, the VTS, and the conserved
linker that joins PI domains, releasing the six PIs, two of
which are chymotrypsin inhibitors and four are trypsin
specific. These peptides are potent inhibitors of trypsin
and chymotrypsin, the major digestive proteinases in
lepidopteran larvae.
The NaPI gene differs from most other PI genes for
insect control tested in transgenic plants because six
individual proteinase inhibitors are produced from a
single NaPI transcript, in contrast to the typical single
or double domain PIs produced from other genes. We
considered that this would allow us to produce higher
levels of PIs in transgenic plants and also introduce the
opportunity to swap one or more of the PI domains in
the multidomain precursor with PIs tailored to the suite
of proteinases in the gut of a target insect.
We began by studying how H. punctigera larvae
respond to ingestion of the six 6 kDa PIs produced
by NaPI. Larvae exposed to both artificial diets and
transgenic tobacco plants containing the PIs were
much smaller and exhibited more mortality than
insects on control diets12. Although we achieved up to
80% mortality, we focused on the 20% of larvae that
appeared unaffected by PI and asked which of the
adaptive procedures we described earlier had been used
by these insects to bypass the toxic effects of the PIs. We
discovered that more than 95% of trypsin activity had
been eliminated in surviving insects, but a chymotrypsin
had been upregulated that was not inhibited by the 6
kDa NaPI chymotrypsin inhibitors10. Thus the insects
had responded by producing a PI resistant protease.
We thought if chymotrypsin activity were essential
for continued protein digestion and hence survival in
the presence of NaPIs, then we needed to find a new
inhibitor that was effective against the PI-resistant
chymotrypsin. We discovered that an inhibitor from
potato inhibitor I class (StPin1A) was an excellent
inhibitor of NaPI-resistant chymotrypsin. Even though
Pin I inhibitors are encoded by a completely different
family of genes, these genes are expressed in the same
tissues as Pin II inhibitors in Solanaceous species and
are also regulated by wounding and methyl jasmonate.
This makes biological sense, since it is likely that the
two families of genes have co-evolved in response to
development of PI-resistant proteases in insects. Thus
it appears that proteinase inhibitors from at least two
distinct classes are needed to provide sustainable insect
protection in a transgenic plant, which explains why
expression of a transgene encoding a single proteinase
inhibitor has failed to provide protection in the past.
We tested this hypothesis by feeding a combination
of NaPI and StPin1A to the closely related and more
damaging species H. armigera. Larval growth was
much more inhibited by the combination of the six 6
kDa NaPI inhibitors and StPin1A compared to the
NaPI or the StPin1A inhibitors alone (Fig. 3). Using
this knowledge we created transgenic cotton plants
expressing both genes and tested them under insect
pressure in the field. Plants expressing both inhibitors
produced 20% more lint than control plants or plants
expressing just the NaPI or the StPin1A gene10.
Mechanism of resistance to NaPI
Even though Lepidopteran larvae have a huge impact
on global food and fiber production, relatively little is
known about the biochemistry of their major digestive
enzymes, the trypsins and chymotrypsins. The features
that enable lepidopteran enzymes to operate at the
highest biological pH ever recorded (pH ~10 – 11)
have not been described, and it is not known why the
activation peptide of insect chymotrypsins is 3 – 4
times longer than chymotrypsins from other species. No
structures are available, and until now the changes that
render some enzymes resistant to inhibition by plant
proteinase inhibitors have not been defined. The nature
of these changes is intriguing because any alterations
in the enzyme that prevent binding of the PIs must not
interfere with normal hydrolysis of proteins, or the
enzyme would be rendered inactive.
To address this question, we looked for the enzyme
responsible for resistance to NaPI13. We exploited the
interaction between NaPI and StPin1A to develop
affinity columns that enabled us to purify the putative
NaPI-resistant chymotrypsin and simultaneously
purify an NaPI-susceptible chymotrypsin. The unique
amino acid sequence from these two proteins allowed
the design of specific primers for cDNA cloning and
sequencing to confirm that the encoded proteins were
indeed chymotrypsins that shared 72% sequence
identity at the amino acid level. The encoded zymogens
were expressed using baculovirus and insect cell
cultures and, after purification from the medium, were
activated with immobilized trypsin.
The putative NaPI-resistant chymotrypsin was not
inhibited by NaPI but was strongly inhibited by StPin1A,
and the NaPI-susceptible chymotrypsin was strongly
inhibited by both PIs, confirming that we had cloned genes
encoding a PI resistant and a PI susceptible chymotrypsin.
We identified the residues that prevented NaPI-binding
to the PI resistant chymotrypsin by substituting residues
in the NaPI-susceptible chymotrypsin with residues from
the NaPI-resistant chymotrypsin. Chymotrypsins with the
alterations were once again expressed using baculovirus
and tested for susceptibility to NaPI and StPin1A inhibitors.
We discovered that substituting the four consecutive amino
acids LANF in the S1’ binding pocket of NaPI-susceptible
chymotrypsin with amino acids VIDL from the NaPIresistant
chymotrypsin converted the NaPI-susceptible
chymotrypsin to a NaPI-resistant chymotrypsin. Molecular
modeling predicted that the F>L substitution had likely
resulted in the loss of several beneficial contacts between
inhibitor and enzyme.
  </p>
  </div>
  </div>
  
</body>
</html>