<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>VIRUS RESISTANT</title>

    <style>


  #navbar {
  background:rgba(20, 80, 25, 5);
  color: #fff;
  overflow: auto;
  font-family: cursive;
  }

  #navbar a {
  color: rgb(221, 221, 233);
  text-decoration: none;
  }

  #navbar h1 {
  float: left;
  padding-top: 10px;
  }

  #navbar ul {
  list-style: none;
  float: right;
  }

  #navbar ul li {
  float: left;
  }

  #navbar ul li a {
  display: block;
  padding: 20px;
  text-align: center;
  }

  #navbar ul li a:hover, 
  #navbar ul li a.current {
    background: rgba(1, 126, 78, 0.26);
  color: rgb(74, 233, 153);
  }

  .container {
  margin: auto;
  max-width: 1100px;
  overflow: auto;
  padding: 0 20px;
  }

  a{text-decoration: none;}

  #main-footer {
  text-align: center;
  background: green;
  color: #fff;
  padding: 20px;
}

.main{
  color: whitesmoke;
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: cursive;
  height: 35rem;
  width: 100%;
}

#scroll{
    width: 70%;
    height: 500px;
    background-color: rgba(221, 191, 133, 0);
    border: 1px solid silver;
    margin-top: 20px;
    box-shadow: 1px 2px 5px 5px greenyellow;
    border-radius: 10px;
    opacity: 1;
    backdrop-filter: blur(15px);
    overflow: hidden;
    overflow-y: scroll;
    position: relative;
    padding: 5px;
}

body{
  background-image: url(../PICS/diseased\ \(9\).jpg);
  background-repeat: no-repeat;
  background-size: cover;
}

     
</style>
</head>
<body>

  <header>
    <nav id="navbar">
      <div class="container">
        <h1 class="logo" alt="Fast Biotechnological Crops"><a href="INDEX.HTML">FBC</a></h1>
        <ul>
          <li><a href="INDEX.HTML">HOME</a> </li>
          <li><a href="GALLERY.HTML">GALLERY</a> </li>
          <li><a href="ABOUT.HTML">ABOUT</a> </li>
        </ul>
      </div>  
    </nav>
  </header>

  <div class="main">
    <div id="scroll">

  <h2>VIRUS RESISTANT</h2>
  <p>These are crops muted of some genes so they become resistant to virus,The introduction and expression of nucleotide sequences of non-viral origin in plants to confer virus resistance were made possible by gene splicing technologies.
    The introduction and expression of nucleotide sequences of viral and
non-viral origin in plants to confer virus resistance were made possible
by gene splicing technologies, development of promoters that ensure
constitutive expression of genes, and advances in techniques that
facilitate transformation of a range of crop plants. It has been nearly 25
years since the first report on the development of tobacco plants resistant
to Tobacco mosaic virus (TMV) (Powell-Abel et al., 1986). Since then
numerous reports have documented the successful production of plants
with durable resistance to a number of different viruses (Baulcombe
1996; Beachy et al., 1990; Dasgupta et al., 2003; Goldbach et al., 2003; Prins
et al., 2008; Sudarshana et al., 2006; Wilson, 1993). We give a brief account
of the anti-viral genes successfully deployed to genetically engineer
plants for virus resistance. A short account of genes that can be used to
silence plant host genes and ways to interfere with virus transmission by
insect vectors are also included.
A. Anti-viral genes of viral origin
1. Protein-mediated resistance
Coat protein genes. The most widely used genes to engineer virus
resistance in plants were derived from coat protein (CP) genes, either as
full length or truncated constructs. The degree of protection conferred by
CP genes varied from near immunity to delay in the expression of overt
symptoms. In some cases, the resistance was broad (Beachy et al., 1990;
Lomonossoff, 1995). All of the currently commercially released virusresistant
crops are developed through incorporation of CP genes. The
molecular mechanisms underlying the CP-mediated resistance are not
fully understood and appear to differ among different viruses (Prins
et al., 2008). Resistance is attributed to viral protein or its RNA or coexistence
of both.
Replication associated proteins. Many single-stranded DNA viruses are
economically important in developing countries. Replication of these
viruses requires interaction between replication associated protein (Rep)
and host polymerases. Rep is indispensable for virus replication (Elmer
et al., 1988). In Section III, examples of the successful production of GEVR
plants to the geminiviruses African cassava mosaic virus (ACMV) and
TYLCV using the Rep approach are provided.
RNA-dependent RNA polymerase genes. Tobacco plants expressing part
of the viral RNA-dependent RNA polymerase (RdRp or replicase)
showed resistance to TMV (Golemboski et al., 1990) and the resistance
was strain specific. Expression of the TMV 54-kDa protein corresponding
to the carboxyl end of the 183-kDa RdRp was essential to confer
resistance (Carr et al., 1992). Similarly expression of full-length 54 kDa
protein or 30% of it was essential to induce resistance to another
tobamovirus, Pepper mild mottle virus (Tenllado et al., 1995). Later studies
attributed this resistance to RNA silencing (Marano and Baulcombe,
1998). Replicase genes derived from potex, poty, alfamo, and cucomoviruses
have been used successfully to induce resistance (Anderson et al.,
1992; Audy et al., 1994; Braun and Hemenway, 1992; Carr et al., 1994;
Wintermantel and Zaitlin, 2000). It is apparent that a high degree of
resistance can be induced to several plant viruses through transfer of
native or truncated forms of replicase genes.
Movement protein genes. Tobacco plants GE with a mutant movement
protein (MP) of TMV blocked not only the movement of TMV, but also
of several other plant viruses including some with a DNA genome
(Lapidot et al., 1993). Interestingly, expression of native MP supported
infection by movement deficient virus strain (Deom et al., 1987).
Movement of potex-, carla-, hordei-, and furoviruses is regulated by a
13-kDa protein, which is part of a triple gene block protein complex.
Expression of mutant forms of the 13 kDa protein conferred resistance
to the homologous virus, as well as some that are dependent on the
triple gene block protein complex for their movement (Seppa¨nen et al.,
1997).
Protease genes. The RNA genome of virus species that belong to the
picornavirus super family is expressed by proteolytic cleavage of a
polyprotein, which is processed by a virus encoded protease (Ryan and
Flint, 1997). For potyviruses, the protease is referred to as NIa (Urcuqui-
Inchima et al., 2001). Plants expressing the viral protease domain of NIa
exhibited a high degree of resistance to virus infection, presumably
through interference of polyprotein processing into functional products
essential for virus replication (Maiti et al., 1993). Integrity of the protease
active site is essential to induce resistance but protease activity alone was
not adequate to elicit protease-mediated resistance in potato to PVY
(Mestre et al., 2003).
2. RNA-mediated resistance
Resistance to viruses in plants can be mediated through RNA silencing
(Prins et al., 2008). Expression of translatable or untranslatable RNA and
anti-sense RNA corresponding to CP cistron of two potyviruses in
tobacco plants conferred a high degree of resistance to mechanical
inoculation as well as infection via aphids (Lindbo and Dougherty, 1992;
van der Vlugt et al., 1992). This type of resistance can also be incorporated
to protect plants against RNA and DNA virus infection through the
expression of short hairpin RNAs (shRNAs) (Lucioli et al., 2003; Prins
et al., 2008). The pathway leading to the generation of virus-specific
small-interfering RNAs (siRNAs) (Hamilton and Baulcombe, 1999) and
systemic silencing have been discussed in Chapter 2.
B. Anti-viral genes of non-viral origin
1. Plant-derived R genes and micro RNAs
Traditionally, plant pathologists, germplasm scientists and breeders have
worked together in identifying sources of plant virus resistance (R) genes
in cultivated and wild species. Wherever possible, R genes have been
bred and introgressed into cultivated varieties. The process can take 5 to
10 years or more before a genotype with desirable traits is made available
for commercial release. Several of the plant virus resistance genes have
been cloned and, as transgenes, have shown resistance to plant viruses.
The N gene from the tobacco variety Samsun NN was the first gene to be
cloned and this gene conferred resistance to TMV in tobacco and tomato
(Whitham et al., 1994, 1996). The majority of virus-specific R genes cloned
so far are single dominant resistance genes and their molecular
architecture is similar to plant resistance genes against bacteria, fungi,
and nematodes (Goldbach et al., 2003; McHale et al., 2006). In addition to
R genes, plants also have recessive resistance genes, which confer
impaired susceptibility to virus infection (Maule et al., 2007). For
additional information on dominant and recessive resistance genes and
their products See Chapters 1 and 4.
Plants also encode microRNAs (miRNAs) of 24 nt in length which
regulate the abundance of cognate mRNAs by guiding their cleavage (Niu
et al., 2006). These miRNAs are generated by processing of pre-miRNA
prescursors by dicer DCL1 in Arabidopsis thaliana (Xie et al., 2004). Niu et al.
(2006) modified a plant miRNA precursor, miR159 from A. thaliana, to
express artificial miRNAs, leading to suppression of viral genes, P69 from
Turnip yellow mosaic virus (TYMV) and HC-Pro from Turnip mosaic virus
(TuMV). Transgenic A. thaliana plants expressing modified miRNAs
showed resistance to TYMVor TuMVand plants expressing both miRNAs
were resistant to the two viruses. Artificial miRNA resistance was found to
be more desirable than that obtained with shRNAs (Qu et al., 2007).
However, it is premature to conclude that this approach is superior to
incorporating viral transgenes.
2. Ribosome-inactivating proteins
Ribosome-inactivating proteins (RIPs), that contain two polypeptide
chains A and B, are catalytic enzymes that deglycosylate a specific base in
the 28S rRNA of eukaryotes. Chain A is the catalytic domain and chain B
binds to galactose. The majority of plant RIPs are type 1 that contain only
chain A. Pokeweed anti-viral protein (PAP) is a RIP known to inhibit
virus multiplication. Transgenic N. tabacum plants that expressed PAP
showed broad-spectrum resistance to several viruses, including
Cucumber mosaic virus (CMV), Potato virus Y (PVY), and Potato virus X
(PVX), when challenged by mechanical inoculation as well as via aphid
transmission (Lodge et al., 1993). This approach has been shown also to
work against a plant DNAvirus (Hong et al., 1996). PAP can degrade 23S
rRNA (Chaddock et al., 1994) and 28S rRNA (Tumer et al., 1997) of
prokaryotes. The mechanism of PAP-mediated resistance to virus
infection was initially thought to be due to inactivation of the 28S rRNA.
However, a deletion mutant of PAP in the C-terminal domain of 28S
rRNA that lacked depurination ability still exhibited virus resistance in
transgenic plants (Tumer et al., 1997). The mechanism underlying the
resistance in GE plants expressing PAP or RIP type 1 is currently not well
understood.
3. Protease inhibitors
Members of several economically important plant viruses that belong to
the families Comoviridae, Potyviridae, and Closteroviridae encode for cis
and/or trans-acting proteases to release individual proteins from a
polyprotein encoded by their genomic RNAs. Protease inhibitors can be
expected to interfere with viral proteases. Tobacco plants expressing a
rice cystatin (a cysteine proteinase inhibitor) were resistant to Tobacco etch
virus (TEV) and PVY but not to TMV (Gutierrez-Campos et al., 1999).
Presumably, the product of the transgene could confer resistance against
those viruses that are dependent on proteases for genome expression and
replication. We are not aware of any report on the successful production
of GEVR food crops using this approach.
C. Anti-viral genes from eukaryotes
1. Expression of dsRNAse and RNA modifying enzymes
Plant RNA viruses and viroids replicate through a dsRNA intermediate.
One of the yeast genes pac 1 encodes for a ribonuclease capable of
degrading dsRNA. Watanabe et al. (1995) isolated this gene from
Shizosaccharomyces pombe and expressed it in N. tabacum cv. Xanthi-nc
plants. These plants showed a decrease in lesion numbers when
challenged by Tomato mosaic virus and a delay in the development of
symptoms when inoculated with CMV and PVY. Further experiments
conducted on potato plants expressing pac 1 demonstrated that Potato
spindle tuber viroid infection was suppressed in these plants, and the
tubers were also free of the viroid (Sano et al., 1997).
In mammals, the interferon system provides a broad-spectrum antiviral
response. During the course of viral infection, interferons induce
synthesis of additional proteins that affect virus multiplication. Among
these, 2u–5u oligoadenylate synthetase (2–5Ase) is a key enzyme which
upon activation by the presence of dsRNA polymerizes ATP to produce
2u–5u oligoadenylates, which in turn activates RNAse L, an endoribonuclease,
that degrades both viral and cellular RNAs. Potato plants
expressing a rat cDNA encoding the 2–5Ase were protected from PVX
infection under field conditions (Truve et al., 1993). Some of the GE potato
lines were found to be superior in resisting virus infection to those
expressing a PVX CP transgene. Expression of a human 2–5Ase in
tobacco also protected plants against CMV and TMV (Ehara et al., 1994).
When tobacco plants with human RNase L were crossed with those GE
with 2–5Ase, offsprings carrying both transgenes were more resistant to
CMV and PVY infection compared to parent GE lines carrying either
RNAse L or 2–5Ase alone (Ogawa et al., 1996).
2. Plantibodies
Ever since the development of antibodies raised against purified plant
virus preparations, expression of antibodies in plants was presumed to
provide protection against virus infection. N. benthamiana containing a
single chain variable fragment (scFv) specific to the CP of Artichoke
mottled crinkle virus showed reduction in disease incidence and delay in
symptom expression (Tavladoraki et al., 1993). Several other reports have
been published on the successful expression of scFvs specific to CPs of
several plant viruses (Prins et al., 2008). A novel approach based on
expressing scFvs to a virus replicase produced plants with broad
resistance to Cucumber necrosis virus and Tomato bushy stunt virus (Family
Tombusviridae; Genus: Tombusvirus), Red clover necrotic mosaic virus
(Family Tombusviridae; Genus: Dianthovirus), and Turnip crinkle virus
(Family Tombusviridae; Genus: Carmovirus) (Boonrod et al., 2004). A logical
extension of this approach would be to test scFvs specific to the GDD box
of RdRp to achieve broad-spectrum protection against plant RNA
viruses. Fomitcheva et al. (2005) expressed MAbs to GDD domains of
RdRp of Barley yellow dwarf virus (BYDV) and NIb of PVY. A scFv clone
capable of detecting both antigens have been identified (Fomitcheva et
al., 2005) but GEVR plants expressing this scFv clone are yet to be
produced.
3. Silencing of host genes involved in virus multiplication
Several host genes are actively involved in virus multiplication.
Mutations of A. thaliana genes TOM1 and TOM3 abolished completely
the multiplication of two strains of TMV (Yamanaka et al., 2002). When
homologs of these genes in N. tabacum cv. Samsun were silenced by RNA
interference, plants were resistant to several tobamoviruses (Asano et al.,
2005). The loss of two host gene products apparently had no effect on the
growth of silenced plants. Interestingly, this resistance was limited to
tobamoviruses, but not to a CMV strain, indicating differences in the
selection of host proteins utilized by viral RdRps. Silencing of TOM1 and
TOM3 homologs in N. benthamiana also prevented tobamovirus multiplication
(Chen et al., 2007). Conceivably this approach can be applied to
any plant virus, given the host genes to be silenced are identified and
characterized. Diaz-Pendo´n et al. (2004) and Maule et al. (2007) have
listed several host factors that influence infection by different viruses,
providing opportunities to apply this strategy.
D. GE resistance to arthropod vectors or transmission of plant
viruses
Another approach to control plant virus spread is through the
development of resistance to insect vectors. GE rice lines expressing a
lecithin gene caused significant reductions in the survival, fecundity, and
development of the brown plant hopper, Nilaparvatha lugens (Rao et al.,
1998). In addition to being a vector of viruses that cause grassy stunt,
ragged stunt, and tungro in rice, N. lugens is a major pest of rice.
Despite a vast amount of research on RNAi-mediated silencing
targeted to viruses this technology is yet to be fully exploited to control
plant virus vectors because of the inability of siRNA made by plants to
trigger silencing in insects. A breakthrough in this regard came recently
when silencing effects of dsRNA in artificial diet containing gossypol
were studied in A. thaliana. siRNA made by plants was unable to trigger
silencing in insects while shRNA was able to turn on silencing in insects
artificially fed on diet spiked with shRNA targeted to insect genes
(Mao et al., 2007). shRNA-mediated silencing in agricultural crops is
yet to be exploited for engineering virus resistance through controlling
vectors.
Another protein of interest is a homolog of the GroEL protein, called
symbionin. This protein is produced by bacterial endosymbionts of
insects, including aphid and whitefly vectors of plant viruses. Symbionin
has the capability to bind virions of persistently transmitted, circulative
viruses. These viruses are dependent on this protein for, their safe
passage through the insect hemolymph and their subsequent secretion
by the salivary glands (Czosnek et al., 2002). A GroEL gene obtained
from whiteflies (Bemisia tabaci) conferred tolerance to TYLCV and
CMV when expressed in transgenic N. benthamiana (Edelbaum et al.,
2009). Tolerance of plants expressing GroEL gene to CMV was unexpected
because the virus is non-persistently transmitted. A GroEL homologue
from B. tabaci was able to bind geminate and icosahedral RNA viruses
(Akad et al., 2004). The potential of GEVR plants expressing GroEL is yet
to be fully exploited.
  </p>
</div>
</div>

</body>
</html>